shared/components/NextCommunityEvent/index.js

```diff
@@ -13,6 +15,26 @@ function googleMapsUrl(location) {
   return `http://www.google.com/maps/place/${latitude},${longitude}`;
 }
 
+export function getHeaderText(startDateTime, endDateTime) {
```

Comment:
Could it be used anywhere else, or is it an implementation detail of this module?

If it's an implementation detail better to make it private.